webextension/background.js

```diff
@@ -202,6 +269,8 @@ const Config = (function() {
         if (!this._uiVariant) {
```

Comment:
hm on this.  I would throw here, I think it's would be a bug at this point.

Reply:
Agreed, but I'm not confident that the addon will remain Shield-driven forever, so I wanted to leave the catch-all there so we can still A/B test in either case. I'll definitely be revisiting this in our next version, though.